README.md

```diff
@@ -15,30 +15,97 @@ And you can check all of our open-source projects at [inaka.github.io](http://in
 
 ## Conventions & Rules
 
-#### Maintain existing style
-> When editing a system written by someone else, stick to the style in which it was designed. If a project has an overall style, stick to that when adding, removing or updating API endpoints as well.
+#### Use snake_case with JSON
```

Comment:
I found 3 misspelled words in this sentence: 
- JSON
	- *Suggested Replacements:* son, jason, jeon
- snake_case
- ####